packages/regl-worldview/src/commands/GLText.js

```diff
@@ -42,6 +42,9 @@ type Props = {
   ...CommonCommandProps,
   children: $ReadOnlyArray<TextMarkerProps>,
   autoBackgroundColor?: boolean,
+  hiresFont?: boolean,
```

Comment:
```suggestion
  highResolutionFont?: boolean,
```
perhaps? We don't have to pay github by the character. 😉

Reply:
@janpaul123 Changed it to highResolutionFont in a recent commit :)